### DISCUSSION TITLE
Setting the Token Source in custom authorizer

### DISCUSSION THREAD

**Discussion by PawelFaron** [serverless/serverless#10545] [Q&A]
Do you maybe know if is there any possibility to set the Token Source in custom authorizer from serverless yml config file?  

![Untitled](https://user-images.githubusercontent.com/42373772/150400386-0174d6ef-04ff-41d8-b0b4-ee7e97d31697.png)

**Comment by pgrzesik** [comment]
Hello @PawelFaron - yes, it's possible to do so via `identitySource`
